### PR TITLE
example outputting to Tenere controller

### PR DESCRIPTION
**Pull request by nargetdev**
I decreased the size of the model (derived from the grid_model(30,30) instantiation)  because I see an index out of bounds exception when it's bigger.  Not sure what that's about.  Also I made the `fixtureToIndices` method of LXFixture public so I could extract the indices when needed from outside.  I think it would probably be a better idea to expose a method of LXFixture to get the indices.

### CODE REVIEW

File: examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java [hunk 1]
```diff
@@ -22,10 +22,7 @@
 import heronarts.lx.LXPattern;
 import heronarts.lx.model.GridModel;
 import heronarts.lx.model.LXModel;
-import heronarts.lx.output.ArtNetDatagram;
-import heronarts.lx.output.FadecandyOutput;
-import heronarts.lx.output.LXDatagramOutput;
-import heronarts.lx.output.OPCOutput;
+import heronarts.lx.output.*;
```
Comment (mcslee) [inline examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java:26]: Are you using an IDE? Better to specifically enumerate only the dependencies that are used. Eclipse or IntelliJ should both do this...

File: examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java [hunk 2]
```diff
@@ -35,16 +32,16 @@
 
   public static LXModel buildModel() {
     // TODO: implement code that loads and builds your model here
-    return new GridModel(30, 30);
+    return new GridModel(10,17);
```
Comment (mcslee) [inline examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java]: Before changing this, I'd love to see what the IndexOutOfBounds exception was that was being caused by 30, 30. That's concerning, implies that some size limit was being improperly hardcoded elsewhere...

File: examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java [hunk 3]
```diff
@@ -55,14 +52,47 @@ public static void addOPCOutput(LX lx) throws Exception {
...
     try {
       LXModel model = buildModel();
       LX lx = new LX(model);
 
+
+      addTenereOutput(lx, "10.200.1.64");
+      addTenereOutput(lx, "10.200.1.65");
+
+      // target some OPC servers
+      String[] controller_ips = { "10.200.1.146", "10.200.1.195", "10.200.1.197"};
```
Comment (mcslee) [inline examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java]: Minor naming convention thing, but should keep the style conventions standard in here. lower_caps_underscore shouldn't be used here, rather should be camelCase.

I'm also not sure these specific IP addresses and Tenere-specific output classes ought to go in the base project, happy to let this slide but let's try to keep project-specific stuff to a minimum in here.

**Review by mcslee — CHANGES_REQUESTED** [review]
Also made a change here so there's a public utility function for getting a list of point indices from any LXFixture, which is LXFixture.Utils.getIndices(LXFixture fixture)

https://github.com/heronarts/LX/commit/01e6a2654c2cd8fc10810a27d0919a0927d64e16

**Comment by mcslee** [comment]
Yo - so it seems like the merge action here has gotten a little weird. Have you rebased your branch against heronarts:master? It looks like all the diffs that I created since are being double-included in the change-set here, rather than it being re-based against an update.

Do you want to do:
git pull --rebase

In your branch from the point that had your commits, and then generate a new clean pull request based upon that?

**Comment by nargetdev** [comment]
Ok yea, I felt like there might be some weirdness – I’ll try again when I have a second.  There should only be like 15 lines of code added to the example, and then the TenereController.java class

**Pull request closed without merging**